.circleci/config.yml

```diff
@@ -121,10 +133,30 @@ jobs:
             conda activate runner-env
             pip install --upgrade cmake
             pip install --upgrade pybind11[global]
+            pip install pybind11-stubgen
             pip install numpy
             pip install twine
       - run:
-          name: Build package
+          name: Build python package
+          command: |
+            eval "$(conda shell.bash hook)"
+            conda activate runner-env
+            DEVELOPER_DIR=$(developer_dir_macos_<< parameters.macos_version >>) \
+              PYPI_RELEASE=1 \
+              CMAKE_BUILD_PARALLEL_LEVEL="" \
+              python setup.py build_ext
```

Comment:
I don't think there is the need for `build_ext` before `install`. Install will take care of building anyway.

Reply:
Yes, thank you for noticing that. It's not necessary for builds that are not in-place. I'm fixing that.